testing/trino-faulttolerant-tests/src/test/java/io/trino/faulttolerant/jdbc/BaseJdbcFailureRecoveryTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.faulttolerant.jdbc;
+
+import io.trino.faulttolerant.BaseFailureRecoveryTest;
+import io.trino.operator.RetryPolicy;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public abstract class BaseJdbcFailureRecoveryTest
+        extends BaseFailureRecoveryTest
+{
+    public BaseJdbcFailureRecoveryTest(RetryPolicy retryPolicy)
+    {
+        super(retryPolicy);
+    }
+
+    @Override
+    protected void createPartitionedLineitemTable(String tableName, List<String> columns, String partitionColumn)
+    {
+    }
+
+    @Override
+    @Test(enabled = false)
+    public void testJoinDynamicFilteringDisabled()
+    {
+    }
+
+    @Override
+    @Test(enabled = false)
+    public void testJoinDynamicFilteringEnabled()
+    {
+    }
+
+    @Override
+    @Test(enabled = false)
```

Comment:
I think it is better to assert on current limitations so we notice that this test needs to be extended when connector becomes more powerful.
I suggest sth like:
```java
    @Override
    public void testAnalyzeTable()
    {
        assertThatThrownBy(super::testAnalyzeTable).hasMessageContaining("This connector does not support analyze");
        throw new SkipException("skipped");
    }
```

(here and in other methods)